src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -13557,13 +13557,10 @@ await using (new T())
 }
 ";
             var comp = CreateCompilation(src, targetFramework: s_targetFrameworkSupportingByRefLikeGenerics, options: TestOptions.ReleaseExe);
-
-            // https://github.com/dotnet/roslyn/issues/72819: The failure is likely unexpected, but is not specific to `allow ref struct` scenario.
-            comp.VerifyDiagnostics(
-                // (36,22): error CS0121: The call is ambiguous between the following methods or properties: 'IMyAsyncDisposable1.DisposeAsync()' and 'IMyAsyncDisposable2.DisposeAsync()'
+            comp.VerifyEmitDiagnostics(
```

Comment:
>comp.VerifyEmitDiagnostics( [](http://example.com/codeflow?start=12&length=27)

I think we should convert this test case to a success case, execute the scenario and observe expected disposal behavior. See, for example, how ```SystemActivatorCreateInstance_03``` takes care of ```CreateInstance``` problem.